docs/install/install-server/install-server.inc

```diff
@@ -61,104 +33,110 @@ Install the software
 
         Please note, the path to the virtual env is arbitrary. Your desired path can override below example.
 
         .. code-block:: sh
 
-            # Install GCC, python3 >= 3.6 and pip
+            # Install GCC, python3 >= 3.9 and pip
             sudo apt-get update
             sudo apt-get install build-essential
             sudo apt-get install python3-pip
 
             # Install wheel and inmanta in a python venv
             sudo apt-get install python3-venv
             sudo python3 -m venv /opt/inmanta
-            sudo /opt/inmanta/bin/pip install wheel
+            sudo /opt/inmanta/bin/pip install -U pip wheel
             sudo /opt/inmanta/bin/pip install inmanta
             sudo /opt/inmanta/bin/inmanta --help
 
             # Install PostgreSQL
             sudo apt-get install postgresql postgresql-client
 
 
-        Download the configuration file named ``inmanta.cfg`` (this name is arbitrary) in your virtual env:
+        Download the configuration files named ``inmanta.cfg`` and ``extensions.cfg`` (these names are arbitrary) in your virtual env:
 
         .. code-block:: sh
 
+            sudo mkdir /opt/inmanta/inmanta.d
             sudo apt-get install wget
             sudo wget -O /opt/inmanta/inmanta.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/inmanta.cfg"
+            sudo wget -O /opt/inmanta/inmanta.d/extensions.cfg "https://raw.githubusercontent.com/inmanta/inmanta-core/master/misc/extensions.cfg"
 
 
-        If you want to use the dashboard you need to install it as well:
+        If you want to use the web-console and/or the dashboard you need to install it as well:
 
-        Get the pre-built package from `our github page <https://github.com/inmanta/inmanta-dashboard/packages/>`_ click on the latest release and then on the right hand side, under ``Assets``, you will see the compressed package. Download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under ``[dashboard]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
+        Get the pre-built package from our `web-console <https://github.com/inmanta/web-console/packages/>`_ and/or `dashboard <https://github.com/inmanta/inmanta-dashboard/packages/>`_ github page. Click on the the package name to go to the package's main page, then on the right hand side under ``Assets``, you will see the compressed package. Download and extract it to your desired directory (preferably, on the same virtual env which was created earlier, in this case, /opt/inmanta). Next, open the ``inmanta.cfg`` file and at the bottom of the file, under the ``[dashboard]`` and/or ``[web-console]`` section, change the ``path`` value to the ``dist`` directory of where you extracted the pre-built package. For instance:
```

Comment:
Do we still need to mention the `dashboard` since the transition to web console is complete?

Reply:
I think it's only 95% complete. Last I checked there was no compile button etc. @bartv could you confirm?

Reply:
yes, I had to use the term `almost` complete. In the blog post, we also stated that it is complete.

Reply:
Oh no, I was just consulting to see if we still need to mention it since quite soon the transition will be 100% complete. As part of the release notes, in the blog, we have already brought up this "completed" transition.  
My suggestion/query was to see if we follow the same footsteps here or not to keep us from modifying the docs again in the near future.

Reply:
Might indeed be best to just mention the web-console. None of our customers install it from source anyway so they still get both.

Reply:
This is OSS though, so our customers don't really have anything to do with this.

Reply:
(the iso docs override this)

Reply:
The compile button was added https://github.com/inmanta/web-console/pull/2502, so we should have that in OSS?

Reply:
All right, I guess in that case we can remove mention of the dashboard.